Makefile

```diff
@@ -62,9 +62,15 @@ $(ASSETS): $(JSONNET_SRC) $(JSONNET_VENDOR) hack/build-jsonnet.sh
 $(JSONNET_VENDOR): jsonnet/jsonnetfile.json
 	cd jsonnet && jb install
 
-generate: clean
+generate: clean $(GOBINDATA_BIN)
```

Comment:
This change should not be here. Note that the target is run in a docker container, but this dependency is installed on the calling host. The make invocation in the target already includes "dependencies" ensuring that all binaries are installed in the docker container.

Reply:
Right. I meant to add this dependency to `pkg/manifests/bindata.go`. That should work, right? Why do we need our homegrown dependency management in that case?